### DISCUSSION TITLE
2.4.40 to 2.4.50 soup upgrade fails with a unknown error 1

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12599] [2.4]
### Version

2.4.40

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

2.4.40 to 2.4.50

New install soup upgrades then wants to run a second time and that is when it fails. the soup log doesn't give any additional context just stating that the was an unknown error 1. Half of the services are then missing so I reverted and everything is running smoothly but unable to upgrade using soup.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Do you have your `/root/soup.log` are there any clues in there?
